_data/projectfooter.yaml

```diff
@@ -17,25 +21,31 @@ links:
         url: https://github.com/quarkusio
   - title: Get Help
     subfolderitems:
+      - page: Forums
+        url: https://groups.google.com/forum/#!forum/quarkus-dev
       - page: Chatroom
         url: https://quarkusio.zulipchat.com
-      - page: Google&nbsp;Groups
-        url: https://groups.google.com/forum/#!forum/quarkus-dev
-      - page: FAQ
-        url: /faq
 
 more_links:
-  - title: Quarkus is proudly made of a number of projects including ...
+  - title: Quarkus is made of community projects
     subfolderitems:
+      - page: Apache Camel
+        url: https://camel.apache.org
+      - page: Eclipse MicroProfile
+        url: https://microprofile.io
+      - page: Eclipse Vert.x
+        url: https://vertx.io/
       - page: Hibernate
         url: https://hibernate.org
       - page: Netty
         url: https://netty.io
       - page: RESTEasy
         url: https://resteasy.github.io
-      - page: Eclipse MicroProfile
-        url: https://microprofile.io
-      - page: Eclipse Vert.x
-        url: https://vertx.io/
-      - page: Apache Camel
-        url: https://camel.apache.org
+  - title: And used in so many more
+    subfolderitems:
+      - page: Thorntail
+        url: https://thorntail.io/
+      - page: WildFly
+        url: https://wildfly.org/
```

Comment:
These two are definitely incorrect. Not sure about the third one.

Reply:
@emmanuelbernard I think it's too early for this part. We don't really have any projects based on Quarkus to put there. Do you agree?